### DISCUSSION TITLE
Send packet captured by wireshark from console by "-n" option

### DISCUSSION THREAD

**Discussion by lazna** [dannagle/PacketSender#384] [General]
is that possible at all? In what format packet should be saved in wireshark?

**Comment by dannagle** [comment]
Something like this has been asked in the past.  It is a really neat idea, but it is not as simple as just echoing out a pcap file. 

Packet Sender cannot recreate accurate timing since Packet Sender operates in userland. I'd have to write a special parser to isolate the data stream and apply a bit of intelligence to segment them appropriately to create Packet Sender format packets. That's as far as it could go. 

However, I agree with those requesting this that even just that much would be helpful.
